src/enums/componentFlags.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * Enum for component validator flags.
+ * enum properties MUST match component ID to scroll correctly.
+ * Order is arbitrary but follow component hierarchy.
+ * */
+export enum ComponentFlags {
+  'articles',
+}
```

Comment:
can you elaborate what this is for?

Reply:
Absolutely!

Reply:
o sorry, i thought you meant in a code comment 😆  There is one there!

To elaborate though, we have a mixin method `validateAndScroll` which takes this static enum. 
This enum is the ID values of components in the UI, so it can identify which component to scroll too when there is an invalid state. 

For example, we create an array of those valid flags `[false]`, and it will reference this enum so it knows which ID to scroll too. 

In this case we've only just started and have one id (flag), but will soon contain all components which have a validity check involved.